src/Microsoft.Dnx.Compilation.CSharp/RoslynProjectReference.cs

```diff
@@ -293,6 +271,26 @@ public DiagnosticResult EmitAssembly(string outputPath)
             }
         }
 
+        private EmitOptions GetEmitOptions()
+        {
+            var emitOptions = new EmitOptions();
+
+            var usePortablePdbString = Environment.GetEnvironmentVariable(EnvironmentNames.PortablePdb);
+
+            // Use portable pdbs if explicitly specified or the platform doesn't support pdb genertion
+            var usePortablePdb = !_supportsPdbGeneration.Value ||
+                                 string.Equals(usePortablePdbString, "true", StringComparison.OrdinalIgnoreCase) ||
+                                 string.Equals(usePortablePdbString, "1", StringComparison.OrdinalIgnoreCase);
```

Comment:
Serious now: we should have a helper for these flag environment variables rather that doing `string.Equals(... true, 1)` etc